kse/src/org/kse/gui/actions/ExamineClipboardAction.java

```diff
@@ -208,9 +208,25 @@ private static byte[] decodeIfBase64(String data) {
         }
         return dataAsBytes;
     }
-
+    
+    private boolean isRedirect(int status) {
+		// normally, 3xx is redirect
+    	if (status != HttpURLConnection.HTTP_OK) {
+			if (status == HttpURLConnection.HTTP_MOVED_TEMP || status == HttpURLConnection.HTTP_MOVED_PERM
+					|| status == HttpURLConnection.HTTP_SEE_OTHER)
+				return true;
+		}
+    	return false;
+    }
+    
     private void downloadCrl(URL url) throws IOException, CryptoException {
-        URLConnection urlConn = url.openConnection();
+    	HttpURLConnection urlConn = (HttpURLConnection) url.openConnection();
```

Comment:
Did you have the possibility to test this with a proxy? I vaguely remember that there was an issue when using HttpURLConnection (ignoring the proxy settings).

Reply:
![image](https://github.com/kaikramer/keystore-explorer/assets/58091322/8279176e-9e8e-4dfb-be4b-c628084886a9)
Works correctly using this type of proxy

Reply:
Ok, I'll do a few tests next week.

Reply:
Worked fine in my tests. Unfortunately I don't have access to the proxy anymore where I have noticed the issues a few years ago.